### PR TITLE
Add Ontology for manufacturing process plannning 

### PR DESCRIPTION
**Pull request by OliverHenrichs**
currently in development by Fraunhofer IPT in Germany;

### CODE REVIEW

**Comment by dgarijo** [comment]
@OliverHenrichs,
the maintainer for this general w3id is Christoph Lange-Beve. Please have him review and agree with these changes.

**Comment by dgarijo** [comment]
@OliverHenrichs any updates on this?

**Comment by dgarijo** [comment]
Closing due to inactivity, please @OliverHenrichs reopen if needed after requesting the permission from the original w3id owner/

File: fraunhofer/mappo/Readme.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Ontology for manufacturing process plannning in the domain of single and small batch production
```
Comment (clange) [inline fraunhofer/mappo/Readme.md:1]: please fix to "planning"

**Review by clange — COMMENTED** [review]
Looks fine to me – hello @OliverHenrichs and @dgarijo from the maintainer.

File: fraunhofer/mappo/Readme.md (hunk 1 above)
Comment (OliverHenrichs) [reply fraunhofer/mappo/Readme.md]: done

**Pull request merged**